hAMRonization/ResFinderIO.py

```diff
@@ -1,20 +1,24 @@
 #!/usr/bin/env python
 
 import json
+import csv
 from .Interfaces import hAMRonizedResultIterator
 
 required_metadata = ['analysis_software_version',
                      'reference_database_version']
 
+optional_metadata = ['input_file_name']
```

Comment:
Given the complexity this metadata change adds to all other parsers, we should just have resfinder3 and resfinder4 as separate parsers